src/app/rugby/[team]/TeamContent.tsx

```diff
@@ -20,17 +20,17 @@ interface TeamContentProps {
 const TeamContent = (props: TeamContentProps) => {
   const { children, players, team, teamSheet } = props
 
-  const [selectedPosition, setSelectedPosition] = useState('')
+  const [selectedPositions, setSelectedPositions] = useState<string[]>([])
```

Comment:
Still a couple of TS errors but logic should be good
![Screenshot 2023-12-28 at 5 57 47 pm](https://github.com/dcarg/teamsheet/assets/39669538/cb64faee-4206-4a20-9cb7-8e3cc0542254)

Reply:
type needs to be updated in `src/app/shared/contexts/teamContext.ts:12`